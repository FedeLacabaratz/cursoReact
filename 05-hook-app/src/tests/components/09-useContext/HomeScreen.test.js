import React from 'react';
import { mount } from 'enzyme';
import HomeScreen from '../../../components/09-useContext/HomeScreen';
import UserContext from '../../../components/09-useContext/UserContext';


describe('Pruebas en el <HomeScreen />', () => {
    
    const user = {
        name: 'Federico',
        email: 'fede@mail.com',
    }

    const wrapper = mount(
        <UserContext.Provider value={{user}}>
            <HomeScreen />
        </UserContext.Provider>
    )

    test('Debe de mostrar el componente correctamente', () => {
        
        expect(wrapper).toMatchSnapshot();
    });
})
