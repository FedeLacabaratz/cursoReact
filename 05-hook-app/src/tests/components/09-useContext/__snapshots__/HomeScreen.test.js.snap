// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Pruebas en el <HomeScreen /> Debe de mostrar el componente correctamente 1`] = `
<div>
  <h1>
    HomeScreen
  </h1>
  <hr />
  <pre>
    {
   "name": "Federico",
   "email": "fede@mail.com"
}
  </pre>
</div>
`;
